### ISSUE TITLE
Control Pause with Foundry Pause

### ISSUE THREAD

**Issue by marcusadmortati** [Octarines/FoundryHourglass#9]
I suggest an option to control hourglass pause synchronized with Foundry pause.

**Comment by Octarines** [comment]
@marcusadmortati just want to make sure I understand your suggested workflow. So are you imagining a tick box or something in the options for a timer that means it doesn't have its own pause, but instead is paused when FoundryVTT is paused?

**Comment by kiloforce** [comment]
I agree.  Sometimes I would like a timer that will run _(such as "hey player make a choice already")_, even if the game is paused.  And sometimes I would like a timer that pauses when the game is paused.  And maybe a 3rd option that pauses when the game is in combat, but advances 6 seconds when combat advances.

**Comment by GambetTV** [comment]
Yeah just to add on to this I would also like an option to have Foundry's Pause button to control the pause function of Hourglass. I can quickly hit the Pause Button in Foundry with Space Key, while I have to hit a very small button on the Hourglass with my mouse, it's just not nearly as quick/intuitive.

**Comment by Octarines** [comment]
Just wanted to let you guys know that I have just dropped a new version of Hourglass (v2.0.0) in which I have implemented a "Sync with Foundry Pause" feature which should do what you have requested. Necro request I know, sorry for that (I've not been doing a lot of role play since my kids appeared), but hopefully you are still enjoying some TTRPGs with timers!

<img width="567" height="728" alt="Image" src="https://github.com/user-attachments/assets/af37fede-c3fd-412a-a9d7-f3c416ffb3d0" />

<img width="807" height="762" alt="Image" src="https://github.com/user-attachments/assets/ebf7ab94-f343-4693-90e7-51555372f577" />
